test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -123,24 +124,30 @@ private String nowPlus(int minutes) {
   @Test
   public void testVerifyColumnNames_found() throws Exception {
     executeUpdate("create table data(id int, other varchar(20))");
-    DatabaseAdaptor.verifyColumnNames(getConnection(),
-        "found", "select * from data", "found", Arrays.asList("id", "other"));
+    try (Connection conn = getConnection()) {
```

Comment:
Why the asymmetry between prepareStatement and getConnection, with the latter not using openObjects? Also, if you had just changed getConnection, couldn't you have avoided stomping on a lot of git history?